### ISSUE TITLE
PullCatch.fn station_code

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [nwfsc-assess/nwfscSurvey#46]
@chantelwetzel-noaa, @melissamonk-NOAA needs station_code for her delta GLM model written in STAN. Do you mind if I add this to the catch pull code? I just wanted to double check that additional columns wouldn't be troublesome for something else downstream.
Thanks.

**Comment by chantelwetzel-noaa** [comment]
@kellijohnson-NOAA I don't think this should cause any issues downstream.  However, just to be safe I would recommend create a branch for these edits.  Does that sound reasonable?

**Comment by kellijohnson-NOAA** [comment]
@melissamonk-NOAA try the station_code branch with
```
library(devtools)
unload(package = "nwfscSurvey")
install_github("nwfsc-assess/nwfscSurvey@station_code")
library("nwfscSurvey")
```

**Comment by chantelwetzel-noaa** [comment]
@kellijohnson-NOAA Can this issue be closed?  If this worked as needed, I think it could also be merged into the main branch.

**Comment by melissamonk-NOAA** [comment]
@kellijohnson-NOAA I honestly don't remember. I agree that starting a new issue is probably easiest.
